### ISSUE TITLE
Consider adding Access-Control-Allow-Origin header to API response

### ISSUE THREAD

**Issue by nevnein** [seven1m/bible_api#26]
When trying to GET a resource with the new fetch() API, the access is blocked due to CORS policy:
> Failed to load https://bible-api.com/Mark%203:10?translation=rccv: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https:/xxx.xxxxx.xx' is therefore not allowed access. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

Please consider adding Access-Control-Allow-Origin: * header to the response.

I am not familiar with Sinatra, else I would've opened a pull request myself; anyway, I found this article which may prove useful: [Adding CORS configuration to a Sinatra app](https://medium.com/addval-labs/adding-cors-configuration-to-a-sinatra-app-1ed426e2c028)
Thank you for your time

**Comment by seven1m** [comment]
I'll consider it!!

We do provide jsonp API that you can use cross-domain: http://bible-api.com/john+3:16?callback=whatever

**Comment by MasterOfTheTiger** [comment]
I can't get it to work. I keep getting the same error.

**Comment by damcclean** [comment]
Yeh, I'm getting the same error here 😢

**Comment by seven1m** [comment]
This took me over a year 😭 but it's finally done!

**Comment by MasterOfTheTiger** [comment]
@seven1m Woo hoo! Thank you so much for this! I might finally use this API for something. :heart:

**Comment by tim-hub** [comment]
I recently got the CORS as as well.

```cess to fetch at 'https://bible-api.com/John+1:1?translation=web' from origin 'xxx' has been blocked by CORS policy: Request header field content-type is not allowed by Access-Control-Allow-Headers in preflight response.
```

**Comment by seven1m** [comment]
@tim-hub I added a new issue for that: #86
